nuheat/api.py

```diff
@@ -63,7 +69,7 @@ def request(self, url, method="GET", data=None, params=None, retry=True):
         :param params: Querystring parameters
         :param retry: Attempt to re-authenticate and retry request if necessary
         """
-        headers = config.REQUEST_HEADERS
+        headers = util.get_request_headers(brand=self._brand)
```

Comment:
if taking my suggestion below, this could be:

```suggestion
        headers = self._request_headers
```